node/forest_libp2p/src/config.rs

```diff
@@ -18,6 +18,8 @@ const DEFAULT_BOOTSTRAP: &[&str] = &[
 pub struct Libp2pConfig {
     pub listening_multiaddr: Multiaddr,
     pub bootstrap_peers: Vec<Multiaddr>,
+    pub mdns: bool,
+    pub kad: bool,
```

Comment:
For consistency? or can just serde rename
```suggestion
    pub kademlia: bool,
```